workflow/scripts/neighborhoods.py

```diff
@@ -299,15 +299,22 @@ def load_enhancers(
     enhancers = run_qnorm(enhancers, qnorm)
     enhancers = compute_activity(enhancers, default_accessibility_feature)
 
+    enhancers[["chr", "start", "end", "name"]].to_csv(
+        os.path.join(outdir, "EnhancerList.bed"),
+        sep="\t",
+        index=False,
+        header=False,
+    )
     enhancers.to_csv(
-        os.path.join(outdir, "EnhancerList.txt"),
+        os.path.join(outdir, "EnhancerList.txt.tmp"),
         sep="\t",
         index=False,
         header=True,
         float_format="%.6f",
     )
-    enhancers[["chr", "start", "end", "name"]].to_csv(
-        os.path.join(outdir, "EnhancerList.bed"), sep="\t", index=False, header=False
+    os.rename(
+        os.path.join(outdir, "EnhancerList.txt.tmp"),
+        os.path.join(outdir, "EnhancerList.txt"),
```

Comment:
Awesome. Did you also need to do this with GeneList.txt?

Reply:
I don't think so. The function that creates GeneList is called before the function that creates EnhancerList and is required for creating EnhancerList (run.neighborhoods.py)